arm-dependencies.sls

```diff
@@ -0,0 +1,78 @@
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+/home/servo/bin:
+  file.directory:
+    - user: servo
+    - group: servo
+    - dir_mode: 755
+    - file_mode: 644
+    
+arm64_links:
+  cmd.wait:
```

Comment:
Do you have an example of such a jinja loop? The complication here is that there's also a file rename going on with the symlink (to change `-linux` to `-unknown-linux`). Sorry I'm such a noob at it, but I couldn't find a good example in the salt docs.